docs/docs/references/user-attributes.mdx

```diff
@@ -167,8 +158,56 @@ models:
             AND ${joined}.sales_region = ${lightdash.attributes.sales_region}
 ```
 
-# Demo: filtering a chart based on user attributes
+## Demo: filtering a chart based on user attributes
 
 The following video gives you a full demo for how to use user attributes to filter chart results.
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/t6QS0rr0PDQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## How user and group attribute values interact
+
+Users can be assigned attributes, but you can also assign groups attributes. So, if a user is assigned an attribute value, but they're also part of a group that's been assigned a value for the same attribute, what happens?
+
+### Column filtering
+
+If the required attributes match **_any_** of the user's group or user attribute values, then the user has access to the column.
+
+For example, if a user is part of a group with the attribute value `kiwi`, another group with the attribute value `orange`, and they've also been assigned as a user to the attribute value `coconut`.
+
+```yaml
+columns: 
+  - name: tropical_fruits_column 
+    meta: 
+      dimension: 
+        required_attributes:  
+          fruits: "coconut"
+```
+
+In this example, the `tropical_fruits_column` will be visible to them because `coconut` is listed in their attribute values `['kiwi','orange', 'coconut']`.
+
+### Row filtering
+
+If the user has group and user attribute values assigned, the user attribute values take precedence over group attribute values. If there are multiple group attribute values, there is no prioritization over which value will be used. 
+
+For example, if a user is part of a group with the attribute value `kiwi`, another group with the attribute value `orange`, and they've also been assigned as a user to the attribute value `coconut`.
+
+```yaml
+models:
+  - name: my_model
+    meta:
+      sql_filter: ${TABLE}.fruit = ${lightdash.attributes.fruit}
+```
+
+**Scenario 1**:
+- The default attribute value is `banana`.
+- The user's been assigned the value `coconut`
+- They also have group values assigned of `kiwi` and `orange` 
+
+The `${lightdash.attributes.fruit}` values above will equal `coconut`. User attribute values take precedence over group attribute values.
```

Comment:
Should we remove the default value from the scenarios as it is not used?

I should replace `values` with `reference` 

```suggestion
The `${lightdash.attributes.fruit}` reference will equal `coconut`. User attribute values take precedence over group attribute values.
```